pages/osx/colima.md

```diff
@@ -0,0 +1,36 @@
+# colima
+
+> Container runtimes for macOS and Linux with minimal setup.
+> More information: <https://github.com/abiosoft/colima>.
+
+- Start Colima:
+
+`colima start`
+
+- Use a config file:
+
+`colima start --edit`
+
+- Start and setup Containerd. Install nerdctl to use Containerd via nerdctl:
+
+`colima start --runtime containerd`
+
+- Start with Kubernetes. `kubectl` is required:
+
+`colima start --kubernetes`
+
+- Customize CPU, memory, disk:
+
+`colima start --cpu {{4}} --memory {{8}} --disk {{100}}`
+
+- Use Docker via Colima:
+
+`brew install docker && colima start && docker run hello-world`
```

Comment:
```suggestion
`brew install docker && colima start`
```
Should we run "hello-world" container?

Reply:
I don't think we should include the `brew install docker` part to stay uniform with the other pages (where we don't document platform-specific install commands) but since it's an `osx` page IG it's fine we could move this as a note to the description instead IMO.